draft-ietf-privacypass-architecture.md

```diff
@@ -439,18 +439,28 @@ trusted device is less than the number of Clients that have solved a CAPTCHA in
 the past day. Attesters SHOULD not admit attestation types that result in small
 anonymity sets.
 
-The trustworthiness of Attesters depends on their ability to correctly and
-reliably perform attestation during the issuance protocol. Indeed, Issuers
-trust Attesters to correctly and reliably perform attestation. However, certain
-types of attestation can vary in value over time, e.g., if the attestation
-process is compromised or maliciously automated. These are considered
-exceptional events and require configuration changes to address the underlying
-cause. For example, if attestation is compromised because of a zero-day exploit
-on compliant devices, then the corresponding attestation procedure should be
-untrusted until the exploit is patched. Addressing changes in attestation
-quality is therefore a deployment-specific task. In Split Attester and Issuer
-deployments (see {{deploy-split}}), Issuers can choose to remove compromised
-Attesters from their trusted set until the compromise is patched.
+Issuers trust Attesters to correctly and reliably perform attestation. However,
+certain types of attestation can vary in value over time, e.g., if the
+attestation procedure is compromised or maliciously automated. Broken
```

Comment:
What does "maliciously automated" mean? You could just remove that:
```suggestion
attestation procedure is compromised. Broken
```

Reply:
This was existing text, but removing it is better!